platform/platform-app/pom.xml

```diff
@@ -348,11 +350,11 @@
         <jackson2-version>2.8.8</jackson2-version>
```

Comment:
`geronimo-jms-spec-version` property is no longer being used in the platform-app camel features. It should be removed from this list